src/vs/workbench/services/issue/browser/issueService.ts

```diff
@@ -70,13 +169,42 @@ export class WebIssueService implements IWorkbenchIssueService {
 	private getIssueUriFromStaticContent(baseUri: string, extension?: IExtensionDescription): string {
 		const issueDescription = `ADD ISSUE DESCRIPTION HERE
 
-Version: ${this.productService.version}
-Commit: ${this.productService.commit ?? 'unknown'}
-User Agent: ${userAgent ?? 'unknown'}
-Embedder: ${this.productService.embedderIdentifier ?? 'unknown'}
-${extension?.version ? `\nExtension version: ${extension.version}` : ''}
-<!-- generated by web issue reporter -->`;
+	Version: ${this.productService.version}
+	Commit: ${this.productService.commit ?? 'unknown'}
+	User Agent: ${userAgent ?? 'unknown'}
+	Embedder: ${this.productService.embedderIdentifier ?? 'unknown'}
+	${extension?.version ? `\nExtension version: ${extension.version}` : ''}
+	<!-- generated by web issue reporter -->`;
```

Comment:
does this indentation show in the final description?

Reply:
ope i think this was eslint or a format on save. it does not show in the final description since it's just a text box description